pom.xml

```diff
@@ -128,6 +128,23 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>tools-jar-profile</id>
```

Comment:
JDKs 9+ have the contents `tools.jar` in new format/file that lives in the lib directory of Java Home. So in those JDKs this still builds fine and in older JDKs it will default to using the `tools.jar`.